yahoo_panoptes/framework/resources.py

```diff
@@ -868,18 +862,22 @@ def setup_resource_cache(self):
         try:
             self._resources_store = PanoptesResourceStore(self._panoptes_context)
         except Exception as e:
-            raise e
+            logger.error('Error while setting up PanoptesResourceStore: %s' % repr(e))
+            raise PanoptesResourceError("Error while setting up resources_store for PanoptesResourceCache")
```

Comment:
This is a good pattern - however, I would suggest creating a new exception class called PanoptesResourceCacheException for these and dropping the phrase 'for PanoptesResourceCache' in the error descriptions.